cforge/commands/settings/profiles.py

```diff
@@ -0,0 +1,259 @@
+                if profile.metadata.environment:
+                    console.print(f"  [dim]Environment:[/dim] {profile.metadata.environment}")
+                if profile.metadata.icon:
+                    console.print(f"  [dim]Icon:[/dim] {profile.metadata.icon}")
+
+    except Exception as e:
+        console.print(f"[red]Error retrieving profile: {str(e)}[/red]")
+        raise typer.Exit(1)
+
+
+def profiles_switch(
+    profile_id: str = typer.Argument(
+        ...,
+        help="Profile ID to switch to. Use 'cforge profiles list' to see available profiles.",
+    ),
+) -> None:
+    """Switch to a different profile.
+
+    Sets the specified profile as the active profile. All subsequent CLI
+    commands will use this profile's API URL for connections.
+
+    Note: This only changes which profile the CLI uses. To fully authenticate
+    and manage profiles, use the Context Forge Desktop app.
+    """
+    console = get_console()
+
+    try:
+        # Check if profile exists
+        profile = get_profile(profile_id)
+        if not profile:
+            console.print(f"[red]Profile not found: {profile_id}[/red]")
+            console.print("[dim]Use 'cforge profiles list' to see available profiles.[/dim]")
+            raise typer.Exit(1)
+
+        # Switch to the profile
+        success = set_active_profile(profile_id)
+        if not success:
+            console.print(f"[red]Failed to switch to profile: {profile_id}[/red]")
+            raise typer.Exit(1)
+
+        console.print(f"[green]✓ Switched to profile:[/green] [cyan]{profile.name}[/cyan]")
+        console.print(f"[dim]Email:[/dim] {profile.email}")
+        console.print(f"[dim]API URL:[/dim] {profile.api_url}")
+
+        # Clear the settings cache so the new profile takes effect
+        get_settings.cache_clear()
+
+        console.print("\n[yellow]Note:[/yellow] Profile switched successfully. " "The CLI will now connect to the selected profile's API URL.")
+
+    except Exception as e:
+        console.print(f"[red]Error switching profile: {str(e)}[/red]")
+        raise typer.Exit(1)
+
+
+def profiles_current() -> None:
+    """Show the currently active profile.
+
+    Displays information about which profile is currently being used by the CLI.
+    """
+    console = get_console()
+
+    try:
+        profile = get_active_profile()
+
+        console.print(f"\n[bold green]Current Profile:[/bold green] [cyan]{profile.name}[/cyan]")
+        console.print(f"[dim]Email:[/dim] {profile.email}")
+        console.print(f"[dim]API URL:[/dim] {profile.api_url}")
+        if profile.metadata and profile.metadata.environment:
+            console.print(f"[dim]Environment:[/dim] {profile.metadata.environment}")
+
+    except Exception as e:
+        console.print(f"[red]Error retrieving current profile: {str(e)}[/red]")
+        raise typer.Exit(1)
+
+
+def profiles_create() -> None:
```

Comment:
We should be able to take a data file like the other create commands so this can be scripted

Reply:
Done